.github/workflows/send-redirects-to-production.yml

```diff
@@ -0,0 +1,52 @@
+
+# Terraform Action to lint and apply updated redirects
+name: AWS Cloud Database Test Execution
+on:
+  workflow_dispatch:
+
+jobs:
+  publish-prod-redirects:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        working-directory: scripts/redirect_creation
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
+
+      - name: create Terraform Tfvars
+        id: tfvars
+        run: |
+          cat << EOF > terraform.auto.tfvars 
+            stage                 = "prod"
+          EOF
+
+      - name: Terraform Format
+        id: fmt
+        run: terraform fmt
+
+      - name: Terraform Init
+        id: init
+        run: terraform init
+
+      - name: Terraform Validate
+        id: validate
+        run: terraform validate -no-color
+
+      - name: Terraform Plan
+        if: github.event_name == 'push'
```

Comment:
is this required?

Reply:
I wanted to check with doc team first to see what their desired trigger is before removing or changing this.